python/ray/data/_internal/stats.py

```diff
@@ -196,6 +197,39 @@ def _get_or_create_stats_actor():
     ).remote()
 
 
+@ray.remote(num_cpus=0)
+class _DatasetMetrics:
+    def __init__(self):
+        self.bytes_spilled = Counter(
+            "ray_data_bytes_spilled",
```

Comment:
same with the metric labels; ideally, we would be keeping track of this mapping between prom metric labels, `_DatasetMetrics` attributes, and other labels/references.